VideoLocker/res/color/nav_btn_selector.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_enabled="true" android:color="@color/edx_brand_primary_accent" />
+    <item android:color="@color/edx_grayscale_neutral_base" />
```

Comment:
Android doesn't expose the [color resources][text color resources] that it uses for the text color selector backgrounds, which _are_ [exposed as theme attributes][text color selector attributes]. 3aa94ec46109bc1ed97090a55a05dc5f6a6845e3 adds an explicit color resource for disabled button text color, which is currently just an alias to `edx_grayscale_neutral_base`.

[text color resources]: https://github.com/android/platform_frameworks_base/blob/lollipop-mr1-release/core/res/res/values/colors_holo.xml#L29

[text color selector attributes]: https://github.com/android/platform_frameworks_base/blob/lollipop-mr1-release/core/res/res/values/themes_holo.xml#L90